File: asistencia/app/Models/Evento.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Evento extends Model
{
    use HasFactory;
    protected $table = 'eventos';


    public $timestamps = false;


    protected $primaryKey = 'id';


    protected $fillable = ['id', 'nom_evento', 'fecha', 'horai', 'min_toler',
    "telefono",'latitud', 'longitud','estado','evaluar','perfil_evento','offline'];
}

File: UPeUDrawerBottonBar/app/src/main/java/pe/edu/upeu/modelo/Evento.kt
package pe.edu.upeu.modelo

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "evento")
data class Evento (
    @PrimaryKey(autoGenerate = true)
    var id:Int,
    var nom_evento:String,
    var fecha:String,
    var horai:String,
    var min_toler:String,
    var latitud:String,
    var longitud:String,
    var estado:String,
    var evaluar:String,
    var perfil_evento:String,
    var offline:String,

    )